src/content/docs/tutorials/crates-tui/app_draw.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Draw
+---
+
+Finally, to render the app, we render a stateful widget using `App` as the state.
+
+```rust
+{{#include @code/crates-tui-tutorial-app/src/app.rs:app_draw}}
+```
+
+To do this, we define a singleton `struct` for the widget:
```

Comment:
Please don't use the word Singleton when describing this. It has some really bad OOP connotations (global mutable state, that is mutated by many disparate places and causes spooky action at a distance).

Reply:
Replaced `singleton` with `unit`.

Reply:
Unit isn't really correct either - that means `()` the type where there is only a single possible value.

Reply:
They are called `unit-like` structs in the official documentation: https://doc.rust-lang.org/book/ch05-01-defining-structs.html#unit-like-structs-without-any-fields

I can change it to say `unit-like`.

Reply:
Done in https://github.com/ratatui-org/website/pull/440/commits/9f9dc1992ef323903a866b90ac0d5f14c187ccf0